components/VM_Arm/src/main.c

```diff
@@ -620,6 +620,29 @@ static void irq_handler(void *data, ps_irq_acknowledge_fn_t acknowledge_fn, void
     }
 }
 
+/* Default RAM initialization. Modules can overwrite this weak function with a
+ * custom initialization. e.g. to use memory shared with other components or
+ * mapped on demand.
+ */
+WEAK int init_ram_module(vm_t *vm)
+{
+    if (0 == ram_size) {
```

Comment:
While I understand the motivation writing comparison in this way, I think it's best to follow the style of existing code. Understanding code is often challenging enough even without mixed style.